React/Base/RCTBridge.m

```diff
@@ -84,8 +84,7 @@ typedef NS_ENUM(NSUInteger, RCTBridgeFields) {
     RCTModuleNamesByID = [NSMutableArray array];
 
     RCTEnumerateClasses(^(__unsafe_unretained Class cls) {
-      if ([cls conformsToProtocol:@protocol(RCTBridgeModule)]) {
-
+      if (class_getClassMethod(cls, @selector(bridgeModule)) != NULL) {
```

Comment:
Is the problem that RubyMotion doesn't let you specify that a class implement a particular Objective-C protcol?